templates/types/streaming/nextjs/package.json

```diff
@@ -29,21 +30,24 @@
     "llamaindex": "0.8.2",
     "lucide-react": "^0.294.0",
     "next": "^14.2.4",
+    "papaparse": "^5.4.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "supports-color": "^8.1.1",
     "tailwind-merge": "^2.1.0",
     "tiktoken": "^1.0.15",
     "uuid": "^9.0.1",
     "vaul": "^0.9.1",
-    "marked": "^14.1.2",
-    "@llamaindex/chat-ui": "0.0.5"
+    "marked": "^14.1.2"
   },
   "devDependencies": {
     "@types/node": "^20.10.3",
     "@types/react": "^18.2.42",
     "@types/react-dom": "^18.2.17",
     "@types/uuid": "^9.0.8",
+    "@llamaindex/core": "^0.4.5",
```

Comment:
How can we make this the same version as in llamaindex?

Reply:
Doesn't @llamaindex/core package already include its dependencies on llamaindex?